src/Datadog.Trace/Tracer.cs

```diff
@@ -217,10 +217,48 @@ public Span StartSpan(string operationName, ISpanContext parent = null, string s
                 span.SetTag(Tags.Env, env);
             }
 
+            // Apply any global tags
+            if (((TraceContext)traceContext).Tracer.Settings.GlobalTags.Count > 0)
```

Comment:
Can't we use `this.Settings` to access the settings here?
```suggestion
            if (Settings.GlobalTags.Count > 0)
```

Reply:
👍